great_expectations/analytics/events.py

```diff
@@ -188,6 +189,17 @@ def __init__(self, checkpoint_id: str | None = None):
         )
 
 
+@dataclass
+class CheckpointRanEvent(_CheckpointEvent):
+    _allowed_actions: ClassVar[List[Action]] = [CHECKPOINT_RAN]
+
+    def __init__(self, checkpoint_id: str | None = None):
+        super().__init__(
+            action=CHECKPOINT_RAN,
+            checkpoint_id=checkpoint_id,
+        )
```

Comment:
Do we want additional values (like the validation definition ids)? Are we aligned with Tetris on this?

Reply:
i'll add them 👍